test/signinfo.cc

```diff
@@ -30,7 +30,7 @@ TEST(Signinfo, ExitToTest) {
   std::vector<SignInfo> signs;
   std::vector<std::string> linguistics;
   std::vector<std::pair<std::string, bool>> default_languages;
-  const std::string linguistic_node_file = "test_linguistic_node.bin";
+  const std::string linguistic_node_file = "test_sign_linguistic_node.bin";
```

Comment:
This file was the cause of the problem.  In the multi-threaded environment the `test_linguistic_node.bin` was getting deleted for the `graphparser.cc` test.

Reply:
so this test also builds to the same directory as `graphparser.cc`? if so, shouldn't we change that?

Reply:
Most are all running in the same directory.  This test is testing specifically signs and is not building data.  This is a temporary file as it is required for `CreateSignInfoList`  We are not testing linguistic node signage here as it is done in the gurka tests.  The real solution is to migrate the old tests to gurka.